### ISSUE TITLE
Continuous Crash on loading of the world

### ISSUE THREAD

**Issue by ds3919** [NovaMachina/ExNihiloSequentia#163]
**Description**
(By the way I am playing All The Mods 6 To The Sky)
So pretty much I was playing and I am not sure what happened but I think it has something to do with fired crucibles because the last thing I was doing was attaching mekanism mechanical pipes to fired crucibles and that is when my game froze and then crashed. Now whenever I try to load up the world, it either crashes on "Joining World" or crashes after getting into the game.

**Version**
Minecraft Version: 1.16.5
Forge Version: 36.0.46 
Ex Nihilo: Sequentia Version: 1.16-1.0.0.2

**Additional context**
Crash Report:  [help.txt](https://github.com/NovaMachina-Mods/ExNihiloSequentia/files/6198619/help.txt)
World File:   [Copy_of_First_World.zip](https://github.com/NovaMachina-Mods/ExNihiloSequentia/files/6200569/Copy_of_First_World.zip)

**Comment by remplerus** [comment]
just 3 sentences to the "pipe bug". 
- I've tested 4 pipe mods and am testing xnet next. 
- I've found out, that only pipez is working perfectly, all other mods (mekanism, refinedpipes and prettypipes (this one i havent figured out, how it is working)) have problems like not pushing the right amount of seeds/saplings/stone into the crucibles and barrels or not even interacting with those (special the wooden crucible).
- So i have just the conclusion that those pipe mods can corrupt any world attatched to any block (not just ex nihilo, i also had a crash with excompressum).
